doc/source/whatsnew/v1.5.1.rst

```diff
@@ -8,7 +8,20 @@ including other versions of pandas.
 
 {{ header }}
 
-.. ---------------------------------------------------------------------------
+.. --------------------------------------------------------------------------
+
+.. _whatsnew_151.enhancements.optional_dependency_management:
```

Comment:
Unfortunatly, I think this should go in 1.6 as a point releases is more related to bug fixes and regressions from a minor release (1.5)